R/precipitation_surplus.R

```diff
@@ -145,10 +145,8 @@ ind_psp <- function(D_PSP,B_LU_BRP){
   dt <- data.table(D_PSP,
                    B_LU_BRP)
   
-  # Separate evaluation for grassland
-  dt[,I_PSP := fifelse(B_LU_BRP %in% OBIC::crops.obic[grepl("gras",crop_name),crop_code],
-                       evaluate_logistic(D_PSP,0.04,180,2),
-                       evaluate_logistic(D_PSP,0.05,300,2.5))]
+  # Calculate indicator score
+  dt[,I_PSP := evaluate_logistic(D_PSP,0.05,300,2.5)]
```

Comment:
I checked how did this in the pilot and I saw that we used the same score for grassland and cropland. So I removed the separate evaluation of grassland and cropland